opcua/common/xmlimporter.py

```diff
@@ -173,8 +173,12 @@ def add_variable(self, obj):
             attrs.ValueRank = obj.rank
         if obj.accesslevel:
             attrs.AccessLevel = obj.accesslevel
+        else:
+            attrs.AccessLevel = ua.AccessLevel.CurrentRead.mask
```

Comment:
For the XML schema it is part of the spec.
Are you sure about make it also default for VariableAttributes() ? It people already used bit masks, it can set an unwanted read access ?